docs/changelog-fragments.d/823.misc.md

```diff
@@ -0,0 +1,4 @@
+Added changelog fragment management infrastructure using [Towncrier]
```

Comment:
@abhinavsingh this is an example change note. Markdown is optional but should be valid if present. Same for the signature.